.github/labeler.yml

```
@@ -20,41 +20,72 @@ module:
           - scripts/**/*
           - temporal/**/*
           - vector/**/*
+database:
+  - changed-files:
+      - any-glob-to-any-file:
+          - db/**/*
+          - lib/db/**/*
 vector:
   - changed-files:
-      - any-glob-to-any-file: vector/**/*
+      - any-glob-to-any-file:
+          - vector/**/*
+          - lib/vector/**/*
 raster:
   - changed-files:
-      - any-glob-to-any-file: raster/**/*
+      - any-glob-to-any-file:
+          - raster/**/*
+          - lib/raster/**/*
+raster3d:
+  - changed-files:
+      - any-glob-to-any-file:
+          - raster3d/**/*
+          - lib/raster3d/**/*
 temporal:
   - changed-files:
-      - any-glob-to-any-file: temporal/**/*
-database:
+      - any-glob-to-any-file:
+          - temporal/**/*
+          - lib/temporal/**/*
+imagery:
   - changed-files:
-      - any-glob-to-any-file: db/**/*
+      - any-glob-to-any-file:
+          - imagery/**/*
+          - lib/imagery/**/*
+
 GUI:
   - changed-files:
-      - any-glob-to-any-file: gui/**/*
+      - any-glob-to-any-file:
+          - gui/**/*
 Windows:
   - changed-files:
-      - any-glob-to-any-file: mswindows/**/*
+      - any-glob-to-any-file:
+          - mswindows/**/*
 macOS:
   - changed-files:
-      - any-glob-to-any-file: macosx/**/*
+      - any-glob-to-any-file:
+          - macosx/**/*
 Linux:
   - changed-files:
-      - any-glob-to-any-file: singularity/**/*
+      - any-glob-to-any-file:
+          - singularity/**/*
 docker:
   - changed-files:
-      - any-glob-to-any-file: docker/**/*
+      - any-glob-to-any-file:
+          - docker/**/*
+          - '**/*Dockerfile*'
+          - '**/*dockerfile*'
 docs:
   - changed-files:
       - any-glob-to-any-file:
           - doc/**/*
           - man/**/*
+          - '**/*.rst'
```

```suggestion
          - '**/*.md'
          - '**/*.rst'
```

I'd also include MD files here.

---

Oh, thanks, I thought I did also!